projects/elife.yaml

```diff
@@ -522,6 +522,33 @@ search:
                     - bus-interviews--{instance}
                     - bus-blog-articles--{instance}
 
+recommendations:
+    domain: False
+    subdomain: recommendations
+    repo: https://github.com/elifesciences/recommendations
+    formula-repo: https://github.com/elifesciences/recommendations-formula
+    formula-dependencies:
+        - https://github.com/elifesciences/builder-base-formula
+        - https://github.com/elifesciences/api-dummy-formula
+    vagrant:
+        ram: 2048
+        ports:
+            1244: 80
+            1245: 8080
+    aws:
+        type: t2.medium # 4 GB of RAM: same as search for now.
+        ports:
+            - 22
+            - 80:
+                cidr-ip: 10.0.2.0/24 # internal access only
```

Comment:
yeah, I think we have a few cases where I've put the very tight `/24` range in there